deployment/v2/utils/updateVanillaGenesis.ts

```diff
@@ -138,6 +146,37 @@ async function updateVanillaGenesis(genesis, chainID, initializeParams) {
             `0x${await zkEVMDB.getBytecode(precalculatedAddressDeployedSovereignBridge)}`)
     }
 
+    // Compute the address of the wrappedTokenImplementation contract deployed by the deployed sovereign bridge with nonce 2
```

Comment:
sanity check bridge implementation has nonce 2

Reply:
Checking nonce is 3